docs/gateway/api-reference/inference-openai-compatible.mdx

```diff
@@ -95,6 +95,43 @@ response = oai.chat.completions.create(
 )
 ```
 
+#### `tensorzero::cache_options`
+
+- **Type:** object
+- **Required:** no (default: `{"enabled": "write_only"}`)
```

Comment:
nit: Maybe don't need to include the default here? Maybe just say "defaults see below" or something like that.